public/global.css

```diff
@@ -1,339 +1,800 @@
-dd ul, dd dl {
-	margin: 0;
+dd ul,
+dd dl {
+  margin: 0;
 }
 dd ul:not(.comma-separated, .no-bullets) {
-	padding-left: 1em;
+  padding-left: 1em;
 }
 
 @media (min-width: 601px) {
-.side-by-side {
-	display: grid;
-	grid-auto-columns: auto;
-	grid-column-gap: 1em;
-	justify-content: stretch;
-}
-.side-by-side:not(.no-margin) {
-	margin-bottom: 1em;
-}
-.side-by-side > * { margin: 0; grid-row: 1; }
+  .side-by-side {
+    display: grid;
+    grid-auto-columns: auto;
+    grid-column-gap: 1em;
+    justify-content: stretch;
+  }
+  .side-by-side:not(.no-margin) {
+    margin-bottom: 1em;
+  }
+  .side-by-side > * {
+    margin: 0;
+    grid-row: 1;
+  }
 }
 
 button.disclosure {
-	background: none;
-	border: none;
-	padding: 0;
-	margin: 0;
-	font-family: inherit;
-	color: hsl(122deg, 44%, 61%);
-	cursor: pointer;
+  background: none;
+  border: none;
+  padding: 0;
+  margin: 0;
+  font-family: inherit;
+  color: hsl(122deg, 44%, 61%);
+  cursor: pointer;
 }
 button.disclosure:hover {
-	text-decoration: underline;
+  text-decoration: underline;
 }
 button.disclosure:active {
-	background: none;
+  background: none;
 }
 button.disclosure:focus {
-	border: none;
+  border: none;
 }
 
 section {
-	background: rgba(255, 255, 255, 5%);
-	padding: 1em;
-	box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2), 0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 1px 3px 0px rgba(0, 0, 0, 0.12);
-	border-radius: 4px;
+  background: rgba(255, 255, 255, 5%);
+  padding: 1em;
+  box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
+    0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 1px 3px 0px rgba(0, 0, 0, 0.12);
+  border-radius: 4px;
 }
 section {
-	margin-bottom: 1em;
+  margin-bottom: 1em;
+}
+section > *:last-child {
+  margin-bottom: 0;
+}
+section > *:first-child {
+  margin-top: 0;
 }
-section > *:last-child { margin-bottom: 0; }
-section > *:first-child { margin-top: 0; }
 section h1 {
-	color: var(--cata-color-dark_gray);
-	text-transform: uppercase;
-	font-size: 1rem;
-	margin: 0;
+  color: var(--cata-color-dark_gray);
+  text-transform: uppercase;
+  font-size: 1rem;
+  margin: 0;
 }
 
 h1:first-child {
-	margin-top: 0;
+  margin-top: 0;
 }
 
 pre {
-	white-space: pre-wrap;
+  white-space: pre-wrap;
 }
 
 :root {
   /*--cata-color-black: rgb(0,0,0);*/
   --cata-color-black: #121212;
-  --cata-color-red: rgb(255,0,0);
-  --cata-color-green: rgb(0,110,0);
-  --cata-color-brown: rgb(97,56,28);
-  --cata-color-blue: rgb(10,10,220);
-  --cata-color-magenta: rgb(139,58,98);
-  --cata-color-cyan: rgb(0,150,180);
-  --cata-color-gray: rgb(150,150,150);
-  --cata-color-dark_gray: rgb(99,99,99);
-  --cata-color-light_red: rgb(255,150,150);
-  --cata-color-light_green: rgb(0,255,0);
-  --cata-color-yellow: rgb(255,255,0);
-  --cata-color-light_blue: rgb(100,100,255);
-  --cata-color-light_magenta: rgb(254,0,254);
-  --cata-color-light_cyan: rgb(0,240,255);
-  --cata-color-white: rgb(255,255,255);
-}
-.c_black { color: var(--cata-color-black); background: var(--cata-color-black); }
```

Comment:
same with this block; this is generated output from the generate-color-css.js script.

perhaps this could get moved to a separate file and .prettierignore'd?

Reply:
How about .prettierignore'ing both global.css and generate-color-css.js for now? They can be refactored later.